frontend/src/pages/detail/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from '@emotion/styled';
+
+import locationIcon from '../../../../common/assets/images/locationIcon.svg';
+import profileImg from '../../../../common/assets/images/profileImg.svg';
+import starIcon from '../../../../common/assets/images/starIcon.svg';
+import CategoryTags from '../../../../common/components/CategoryTags/CategoryTags';
+import TextWithIcon from '../../../../common/components/TextWithIcon/TextWithIcon';
+
+function Profile() {
+  return (
+    <StyledContainer>
+      <StyledProfileImg
+        src={profileImg}
+        alt="트레이너 이미지"
+      ></StyledProfileImg>
```

Comment:
p3: self-closing 하면 될 것 같아 ㅎㅎ 👍
```suggestion
      <StyledProfileImg
        src={profileImg}
        alt="트레이너 이미지"
      />
```